library/common/engine.cc

```diff
@@ -57,6 +57,7 @@ envoy_status_t Engine::run(const std::string config, const std::string log_level
         Envoy::Server::ServerLifecycleNotifier::Stage::PostInit, [this]() -> void {
           server_ = TS_UNCHECKED_READ(main_common_)->server();
           client_scope_ = server_->serverFactoryContext().scope().createScope("pulse.");
+          stat_name_set_ = client_scope_->symbolTable().makeSet("pulse");
```

Comment:
for context: https://docs.google.com/document/d/1PHvqLEFwiV062RRIMJ2JUd492ZJABOwH-vdK0H-oIRw/edit?disco=AAAALqBrykI